packages/analytics-js/src/services/StoreManager/StoreManager.ts

```diff
@@ -111,7 +112,10 @@ class StoreManager implements IStoreManager {
   initializeStorageState() {
     const globalStorageType = state.storage.type.value;
     let trulyAnonymousTracking = true;
-    const entries = state.loadOptions.value.storage?.entries;
+    const entries = mergeDeepRight(
+      state.loadOptions.value.storage?.entries ?? {},
+      state.consents.postConsent.value.storage?.entries ?? {},
+    );
```

Comment:
This is not right. Say in the post consent state global storage type is `in-memory` and in the load option storage entries `userTraits` type is local storage and in post consent storage entries `userTraits` is not provided.
In this case, the final storage type of userTraits will be local storage not in-memory.

Reply:
Done.